helm_deploy/hmpps-book-a-prison-visit-ui/values.yaml

```diff
@@ -14,6 +14,8 @@ generic-service:
     enabled: true
     host: app-hostname.local    # override per environment
     tlsSecretName: hmpps-book-a-prison-visit-ui-cert
+    annotations:
+      nginx.ingress.kubernetes.io/limit-rpm: "120"
```

Comment:
Adding rate limiting here, using same config as HwPV external as a starting point. Have dismissed the code scanning alerts about missing limiting on the basis that this is now in place.